cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -95,7 +59,7 @@ def _init_service():
 
     # rabbitmq restart exits with 143 status code that is valid in this case.
     systemd.restart(RABBITMQ, ignore_failure=True)
-    wait_for_port(INSECURE_PORT)
+    wait_for_port(SECURE_PORT)
 
     deploy(
         src=join(CONFIG_PATH, 'rabbitmq.config'),
```

Comment:
`rabbitmq.config` isn't even rendered - can we deploy it in the rpm? it'd need to somehow overwrite the rpm from the upstream.

Reply:
and if we make removing the mnesia file part of `%post` in the rpm then we'd be able to get rid of this whole function (one less rabbitmq restart in bootstrap!)

Reply:
Didn't notice this comment before... re. removing data in `%postrm` I don't think we should, & I think long as it's easy & sane (i.e. only stuff we might expect) the install/configure step should be able to cope with a dirty environment